Dockerfile

```diff
@@ -22,65 +24,69 @@ RUN go build -tags beta .
 FROM ${BRANCH} AS builder
 RUN echo "Build ${BRANCH} version"
 
-# Second stage. Creating an image
-# -----------------------------------------------------------------------------
+# Second stage: Create the final image
 FROM alpine:3.18
 
-ARG BUILD_DATE
-ARG VCS_REF
 ARG THREADFIN_PORT=34400
-ARG THREADFIN_VERSION
 
-LABEL org.label-schema.build-date="${BUILD_DATE}" \
-      org.label-schema.name="Threadfin" \
+LABEL org.label-schema.name="Threadfin" \
       org.label-schema.description="Dockerized Threadfin" \
       org.label-schema.url="https://hub.docker.com/r/marcelGoerentz/threadfin/" \
-      org.label-schema.vcs-ref="${VCS_REF}" \
       org.label-schema.vcs-url="https://github.com/marcelGoerentz/Threadfin" \
-      org.label-schema.vendor="Threadfin" \
-      org.label-schema.version="${THREADFIN_VERSION}" \
-      org.label-schema.schema-version="1.0" \
-      DISCORD_URL="https://discord.gg/hrqg9tgcMZ"
+      org.label-schema.vendor="Threadfin"
 
+# Environment Variables
 ENV THREADFIN_BIN=/home/threadfin/bin
 ENV THREADFIN_CONF=/home/threadfin/conf
 ENV THREADFIN_HOME=/home/threadfin
 ENV THREADFIN_TEMP=/tmp/threadfin
 ENV THREADFIN_CACHE=/home/threadfin/cache
-ENV THREADFIN_UID=31337
-ENV THREADFIN_GID=31337
 ENV THREADFIN_USER=threadfin
 ENV THREADFIN_DEBUG=0
 ENV THREADFIN_PORT=${THREADFIN_PORT}
 ENV THREADFIN_LOG=/var/log/threadfin.log
 
+# Default UID/GID (can be overridden by environment variables)
+ENV PUID=31337
+ENV PGID=31337
```

Comment:
Shouldn't we change them to 1000?
And leave it to the user to change them if necessary?

Reply:
@marcelGoerentz I'm pretty sure threadfin needs it like that for some reason... originally tried the 1000 and then it wont loadup. Decided to leave it as is and just add the threadfin user to the USERS group etc.

Reply:
@gordlaben I will cross check this

Reply:
So after my check, the normal container starts up without an issue.
I will check with one of your compose files to see if there is an issue.